### DISCUSSION TITLE
Import IPAM including interfaces

### DISCUSSION THREAD

**Discussion by flinky50** [netbox-community/netbox#10538] [Q&A] [answered]
Hi There,

Does anyone know the correct syntax to use when importing IPAM using a CSV form. I'm constantly getting erros with the interface part. I've created multiple devices and added interfaces to each but can't figure out how to do it. 

EG I've created device called Switch01 and on the switch I've created interfaces named "Port 1" all the way through to "Port 48"

Trying to import over 500 addresses so would much rather just upload 1 file then manually enter this

**Comment by candlerb** [comment]
Can you show one example line which is failing, including both the header line and the data line?

**Comment by flinky50** [comment]
So basically everything else works except the interface field. I have a device called AU-W-DAN-HO-SW07 and have created 52 interfaces on the device. Just trying to figure out the correct syntax to import it to port 20 on this specific device.

![image](https://user-images.githubusercontent.com/114894442/193689108-c48eb7aa-e512-4b03-8464-1e3b0b62e55b.png)

**Comment by cyberndj** [comment, marked as answer]
My 2c...
It looks like your missing device in your header row and a comma in your data row after the device and before Port 20.
ie. like
address,status,description,device,interface
10.150.65.200/24,active,Test Server,AU-W-DAN-HO-SW07,Port 20

**Comment by flinky50** [comment]
Oh wow can't believe I missed that. Sometimes takes another set of eyes

Thank you very much working now
